devex/vsextension/ProjectWizard/VCTargets/1033/openenclave-general.xml

```diff
@@ -35,6 +35,16 @@
                   Category="General">
   </StringProperty>
 
+  <EnumProperty Name="CryptoLib"
+                DisplayName="Crypto Library"
+                Description="Crypto Library to link with."
+                Category="General"
+                IsRequired="true"
+                Default="mbedtls">
+    <EnumValue Name="mbedtls" DisplayName="mbedTLS" />
```

Comment:
FYI, since the donation to TrustedFirmware org, they've been more consistent in using the brand name as "Mbed TLS" in their docs:

```suggestion
    <EnumValue Name="mbedtls" DisplayName="Mbed TLS" />
```

Reply:
Done